student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,10 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    const char* hard_name = my_username();
+    const char* usernametxt_name = malloc_username_from_conf_file();
```

Comment:
@nihalthirunakarasu what happens to the string from `usernametxt_name` after returning from this function?